tsfile/src/main/java/org/apache/iotdb/tsfile/read/filter/basic/UnaryFilter.java

```diff
@@ -51,4 +60,36 @@ public FilterType getFilterType() {
 
   @Override
   public abstract Filter clone();
+
+  @Override
+  public void serialize(DataOutputStream outputStream) {
+    try {
+      outputStream.write(getSerializeId().ordinal());
+      outputStream.write(filterType.ordinal());
+      ReadWriteIOUtils.writeObject(value, outputStream);
+    } catch (IOException ignored) {
+      // ignored
```

Comment:
But I don't think there was no chance to throw an Exception even it sounds safe technically.  In case it has an exception,  it'll be hopeless to locate the bug even if it's a bug from JDK.